packages/main/src/Calendar.js

```diff
@@ -61,26 +61,56 @@ const metadata = {
 			multiple: true,
 		},
 
+		/**
+		 * Determines the мinimum date available for selection.
+		 *
+		 * @type {Object}
+		 * @defaultvalue undefined
+		 * @public
```

Comment:
add @since tags 1.0.0-rc.6 for all new public properties

Reply:
Will do the change with the changes for day-picker.

Reply:
done